src/backend/distributed/deparser/citus_ruleutils.c

```diff
@@ -876,138 +875,6 @@ pg_get_indexclusterdef_string(Oid indexRelationId)
 }
 
 
-/*
- * pg_get_table_grants returns a list of sql statements which recreate the
- * permissions for a specific table.
- *
- * This function is modeled after aclexplode(), don't change too heavily.
- */
-List *
-pg_get_table_grants(Oid relationId)
```

Comment:
this is used in enterprise

Reply:
these would be caught by this PR: https://github.com/citusdata/citus/pull/4027 assuming the compilation would fail on enterprise with the removal of this function

Reply:
> This is used in enterprise, but it might make sense to move it there to avoid confusion.

Then I should remove all the functions used in enterprise but not used in community